test/mock/demo-login-consent-server/templates/login.html

```diff
@@ -58,15 +57,27 @@
                             <input class="w-full text-gray-700 leading-tight inputColor focus:outline-none"
                     id="password" name='password' type="password" value="f00B@r!23">
                         </div>
-
                         <div class="flex items-center justify-center">
-                            <button class="gradient text-sm sm:text-xs rounded-md text-white mb-8 h-10 px-8 shadow-lg" type='submit' id="accept" name='btn_login'>
-                                Sign In
+                            <button class="flex items-center justify-center gradient rounded-md mb-8 h-10 px-8 shadow-lg"
+                                    type='submit'
+                                    id="accept" onclick="signIn()">
+                                <span class="flex-0" id="btn_text"> Sign in </span>
```

Comment:
Can we remove extra spaces around the text?
```suggestion
                                <span class="flex-0" id="btn_text">Sign in</span>
```